### PR TITLE
Add Flake8 check plugin to pytest. Fix code style

### PR DESCRIPTION
**Pull request by mvoitko**

### CODE REVIEW

**Review by tswast — APPROVED** [review]
Thanks for doing this!

Can you share the command-line output of running this lint test locally?

**Comment by mvoitko** [comment]
@tswast there is no output for flake8 part when there is no issues.

**Comment by mvoitko** [comment]
@tswast Could you please merge?

File: test/test_sqlalchemy_bigquery.py [hunk 1]
```diff
@@ -345,7 +346,10 @@ def test_session_query(session, table, session_using_test_dataset, table_using_t
                 table.c.string,
                 col_concat,
                 func.avg(table.c.integer),
-                func.sum(case([(table.c.boolean == True, 1)], else_=0))
```
Comment (tswast) [inline test/test_sqlalchemy_bigquery.py:348]: FYI: I was getting a test failure for "is True". Changing to compare with equality and a sqlalchemy literal fixed the test case.

Comment (mvoitko) [reply test/test_sqlalchemy_bigquery.py:348]: Thanks a lot!

**Pull request merged**